src/pages/index.tsx

```diff
@@ -34,6 +35,16 @@ const HomePage: NextPage<HomePageProps> = ({ meta, allCases, recentCases }) => {
             deadCases={deadCases.length}
           />
         </Section>
+        <Section title="Mapa">
+          <CasesMap
+            // temp data while get CMS/graphql updated
+            sampleData={[
+              { ...recentCases[0], lat: 10.0274898, lng: -84.2224415 },
```

Comment:
this prop needs to go away once lat and lng start to come from cases graphql query response